spec/classes/software_social_skype_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe 'software::social::skype' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) do
+        facts
+      end
+
+      context 'with defaults' do
+        if facts[:operatingsystem] == 'Darwin'
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_package(/Skype-.+/).with_provider('appdmg') }
```

Comment:
This step [is failing](https://travis-ci.org/edestecd/puppet-software/builds/290355530#L1112), too. I need to cover a [variable version string and the provider argument](https://github.com/edestecd/puppet-software/blob/master/manifests/social/skype.pp#L19-L23) (the regex and `with_provider` should do it).

@edestecd Can you suggest what could be the problem here? -- I've tried `with_provider(appdmg)` without quotes without success.

Reply:
I'm not sure. Have you tried it without the .with_provider ? If that still does not work, I might suspect the regex. I can't find any examples of using a regex in a resource name like that. You may have to hard code the version.

Reply:
It's obviously the regex. So, I have to hardcode the version. :worried: